dashboard/app/views/layouts/_footer.html.haml

```diff
@@ -38,19 +38,4 @@
           %br/
           !="&copy; Code.org, #{Time.now.year}"
 
-- is_not_cached_script = !@script || !Gatekeeper.allows('public_caching_for_script', where: {script_name: @script.name})
-- if DCDO.get('hoc_secret', nil) && is_not_cached_script
```

Comment:
This logic was unnecessarily complicated and I simplified it.  It's fine to write the hoc secret regardless of what the public caching settings are.